contracts/common/lib/Math256.sol

```diff
@@ -33,6 +33,7 @@ library Math256 {
     /// This differs from standard division with `/` in that it rounds up instead
     /// of rounding down.
     function ceilDiv(uint256 a, uint256 b) internal pure returns (uint256) {
+        require(b != 0, "Division by modulo by 0");
```

Comment:
Sorry, we are currently in a frozen state for the V2 release (accepting only significant bug fixes).
I'd propose not modifying the contracts whenever if possible :pray:

Reply:
no worries, I can revert that expectation.  Sorry I've been slower, this has been a free time project for contribution.

Reply:
Thank you, that would be great!